zeth_tool/prove_cmd.cpp

```diff
@@ -71,7 +75,8 @@ class prove_cmd : public generic_subcommand<prove_cmd>
         options.add_options()(
             "primary_inputs,p",
             po::value<uint16_t>(),
-            "Number of primary inputs (default: 1)");
+            "Number of primary inputs (default: 1)")(
+            "profile,p", "Enable profiling output");
```

Comment:
Don't you have a clash in the "short flags" names here? (i.e. `p` is used for both the short flag of `profiling` and `primary inputs`)?

Reply:
Ah yes.  Thanks.  Switched to `r` for profile.

Reply:
Thanks